### ISSUE TITLE
In negative layers of bug protection and cannot be edited or removed

### ISSUE THREAD

**Issue by thiagogebrimm** [GriefPrevention/GriefPrevention#1431]
### What happened:
In negative layers of bug protection and cannot be edited or removed

### What was expected:
That has how to remove / edit negative layer claim

### Steps to reproduce:
__This may include a build schematic, a video, or detailed instructions to help reconstruct the issue__
1. Try to edit claim in negative layer
2. Does not work

### Server and GriefPrevention version:
___This can be found by running `/version` and `/version GriefPrevention` on your server.___
git-Purpur-1190 (MC: 1.17)
GriefPrevention 16.17.1 

### Stack trace/error or server log

```
Paste entire stack trace right here inside these backticks.

```
Or, paste the server.log at gist.github.com and paste the link here: 

### GriefPrevention config.yml

```yml

Paste the config.yml right here inside these backticks. Please do not use a paste service for this.

```

### Debugging steps attempted

- [ ] (If applicable) I attempted running the server without GriefPrevention installed
  - [ ] (If applicable) The problem does _not_ occur if GriefPrevention is removed from the server.
- [x ] I attempted running only GriefPrevention on the server
  - [ ] Issue still occurs even if GriefPrevention is the only plugin running
- [ ] I attempted testing for the issue on a new server
  - [ ] Issue still occurs even after testing on a new server
- [x] I attempted testing for the issue with the vanilla client (no mods)
  - [ ] Issue still occurs even after testing with the vanilla Minecraft client

**Comment by Spoonerj1** [comment]
I loaded up a test server and can confirm that a claim will be created, but then it does not protect anything below y0. When using the stick to inspect claims it highlights the area at eye level for the player, which does not resume "normal" behavior until reaching y0. At this point the claim is highlighted at the highest block (which is what I see normally) 

I guess technically the stick is mostly working to highlight the claim, but the claim is still stopping at y0 even when created at y -1 or lower. 

I created several claims in 2 test worlds at y -15 and y -20. 
I also made some claims at y 64 and had no trouble with those. 

Plugin list (if applicable):
datapack (attached) from mojang for larger world height
[CavesAndCliffsPreview.zip](https://github.com/TechFortress/GriefPrevention/files/6745542/CavesAndCliffsPreview.zip)
GriefPrevention 16.17.1-2b69191
Paper:  
[22:39:29 INFO]: This server is running Paper version git-Paper-66 (MC: 1.17) (Implementing API version 1.17-R0.1-SNAPSHOT) (Git: 31d8fd5)

Debugging steps attempted

(If applicable) I attempted running the server without GriefPrevention installed
N/A   

(If applicable) The problem does not occur if GriefPrevention is removed from the server.
N/A


[x] I attempted running only GriefPrevention on the server

[x] Issue still occurs even if GriefPrevention is the only plugin running

[x] I attempted testing for the issue on a new server

[x] Issue still occurs even after testing on a new server

[x] I attempted testing for the issue with the vanilla client (no mods)

[x] Issue still occurs even after testing with the vanilla Minecraft client


EDIT:: I dont have a server log with just GP installed, but I could make one if needed
